src/pedido.py

```diff
@@ -0,0 +1,41 @@
+"""
+Este módulo define las clases relacionadas con los pedidos,
+incluyendo sus estados y los detalles del pedido.
+"""
+
+from dataclasses import dataclass
+from enum import Enum
+from pieza import PiezaInventario
+from sede import Sede
+
+@dataclass(frozen=True)
+class Pedido:
+    """
+    Representa un pedido de piezas entre dos sedes como un objeto inmutable.
+
+    Atributos:
+        piezas_requeridas (dict[Pieza, int]): Diccionario de piezas requeridas y sus cantidades.
+        sede_origen (Sede): Sede desde donde se envían las piezas.
+        sede_destino (Sede): Sede a la que se envían las piezas.
+        tiempo_estimado_llegada (float): Tiempo estimado para que la pieza llegue a la sede destino (en horas).
+        costo_transporte (float): Costo del traslado basado en distancia
+    """
+    piezas_requeridas: dict[PiezaInventario, int]
+    sede_origen: Sede
```

Comment:
Debéis tratar de entender el concepto de Única Fuente de Verdad. Es parecido a lo de la integridad referencial de bases de datos. Básicamente lo que viene a decir es que no podéis tener múltiples copias de la misma información en muchos sitios. Aquí estás metiendo la sede, que tiene un montón de piezas de inventario, en cada uno de los pedidos. Aparte del problema de duplicación de contenidos, el problema es que puede tener valores diferentes en sitios diferentes y que no tienes una sola fuente de verdad.

Reply:
Entiendo, toda la razón. En ese caso usaré el nombre de la sede para identificar la sede origen y la sede destino para realizar el pedido, y así me evito tener múltiples copias de las piezas lo cual haría el proceso lento y redundante, y por ende una Única Fuente de Verdad